server/build.sbt

```diff
@@ -7,10 +7,22 @@ scalaVersion := "2.13.8"
 idePackagePrefix := Some("fr.dopolytech.polydrive")
 
 val AkkaVersion = "2.6.18"
-libraryDependencies += "com.typesafe.akka" %% "akka-actor-testkit-typed" % AkkaVersion % Test
-libraryDependencies +=
-  "com.typesafe.akka" %% "akka-actor-typed" % AkkaVersion
-libraryDependencies += "com.typesafe.akka" %% "akka-stream-typed" % AkkaVersion
-libraryDependencies += "com.typesafe.akka" %% "akka-discovery" % AkkaVersion
+
+libraryDependencies ++= {
+  lazy val minioVersion = "8.3.7"
+  lazy val commonsioVersion = "20030203.000550"
```

Comment:
Not useful to have `lazy` in build.sbt

Also move it above where `AkkaVersion` stands